### DISCUSSION TITLE
Forced Google auth for visitors

### DISCUSSION THREAD

**Discussion by MarcelVolosin** [LiveHelperChat/livehelperchat#2141] [Q&A] [answered]
Hi,
is there a option/plugin to include forced Google auth login for visitors?
So that users will be forced to login to their google account before being able to send first chat message.
Thank you very much for your time.

**Comment by remdex** [comment, marked as answer]
Hi,
At the moment not :)

**Reply by MarcelVolosin** [reply]
thanks a lot

**Comment by link89** [comment]
Hi @MarcelVolosin Does that mean currently there is no way to force user to login before they can start a chat? Or is it possible to ask user to login during the chat to escalate the support level?

**Reply by MarcelVolosin** [reply]
According to author, it is not possible using google auth.
Maybe there is some other authentication method, but I did not investigate further since I needed google.

**Reply by remdex** [reply]
No, not possible to force login visitors. P.s you can always order paid support for it being implemented
